pkg/activator/activator.go

```diff
@@ -267,6 +265,6 @@ func (a *activator) removeReadinessTracker(obj interface{}) {
 	}
 
 	a.trackersMux.Lock()
+	defer a.trackersMux.Unlock()
```

Comment:
If you'd like, there are a few other places in this file where we can use this pattern (search for `.Lock()`)